homatransport/src/common/NicBase.ned

```diff
@@ -0,0 +1,81 @@
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see http://www.gnu.org/licenses/.
+//
+
+package homatransport.common;
+
+import inet.node.inet.NodeBase;
+import inet.applications.ITCPApp;
+import inet.applications.IUDPApp;
+import inet.transportlayer.ITCP;
+import inet.transportlayer.IUDP;
+import homatransport.application.ITrafficGeneratorApp;
+import homatransport.transport.ITransportScheme;
+
+//
+// A simplified version of StandardHost module from inet package. Many
```

Comment:
More comments on this please. What  is included in this module, the major parts, how its being used, etc.